pangram.py

```diff
@@ -0,0 +1,40 @@
+# Solution 3 - O^n2 Quadratic Time
```

Comment:
I'd be a little careful about calling this O(n^2). What is n in this case? If it is the length of the string, it's not O(n^2)... perhaps you want to say that it is O(n * m), where m is the length of the alphabet (26 in this case). Arguably, this is better called linear than quadratic, since 26 is constant.